teacher_class.rb

```diff
@@ -0,0 +1,16 @@
+require './person_class'
+
+class Teacher < Person
+  def initialize(specialization, name = 'Unknown', parent_permission: true)
+    super(30, name, parent_permission)
+    @specialization = specialization
+  end

```

Comment:
- [ ] This `Constructor extends parent's constructar by adding @specialization` like so: 
```ruby
  def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end
```
Kindly remove the hardcoded value of age.